src/calendar_widget/Calendar_Widget.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, window_name, pos_x=0, pos_y=0, size=300, style=None, command=
             heading_inc = heading_inc + 1
 
         font_size = int((box_depth * 0.75) * 0.75)
-        weekday_font = font.Font(family=weekday_font_family, size=font_size)
+        weekday_font = font.Font(family=weekday_font_family, size=font_size, weight="wholesystem-font-weight", slant="wholesystem-font-style")
```

Comment:
You cant pass the variable if you use quotes.
weight="wholesystem_font_weight" ---incorrect 
weight=wholesystem_font_weight    ---correct